### PR TITLE
7.0.1 bugfix

### PR DESCRIPTION
**Pull request by jdogmcsteezy**
- Updated the Copyright
- Fixed issue with displaying the incorrect tool version

### CODE REVIEW

File: src/include/asql_conf.h [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019-2021 Aerospike, Inc.
+ * Copyright 2015-2022 Aerospike, Inc.
```
Comment (ClaytonKnittel) [inline src/include/asql_conf.h:2]: Was this supposed to be changed from 2019 to 2015?

Comment (jdogmcsteezy) [reply src/include/asql_conf.h:2]: I was curious if it mattered

Comment (ClaytonKnittel) [reply src/include/asql_conf.h:2]: Probably not, but if that file wasn't added until 2019 maybe keep it there just for transparency

Comment (jdogmcsteezy) [inline src/include/asql_conf.h:2]:
```suggestion
 * Copyright 2019-2022 Aerospike, Inc.
```

**Review by ClaytonKnittel — APPROVED** [review]

**Pull request merged**